decidim/decidim
# Meetings iframe visibility

#### :tophat: What? Why?

This PR:

* Adds a new enum attribute to meetings to control level access to iframe
* Adds management in meetings admin panel to select iframe access level when iframe is enabled for meeting
* Changes online meeting cell visibility considering:
  * Iframe access level of meeting
  * Privacy and transparency of meeting

#### :pushpin: Related Issues

- Fixes #7719 
- Fixes #7720 

#### Testing

1. Sign in as admin
2. In the dashboard, go to edit a meeting form in a process
3. Select "Online" as meeting type
4. See that you have the options "Iframe embed type" and "Iframe access level". They should work as described at #7719 


#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots

![image](https://user-images.githubusercontent.com/717367/140710580-d00af677-b480-4080-a33b-be0353c4858e.png)


#### Embed type: embed in meeting page

![image](https://user-images.githubusercontent.com/717367/140710658-b18fb54e-4a28-49a2-bdf7-50b11ba675ca.png)


#### Embed type: live event

![image](https://user-images.githubusercontent.com/717367/140710689-7d43c9dc-365e-4b27-b0c6-242fd4fbc2f2.png)


#### Embed type: external

![image](https://user-images.githubusercontent.com/717367/140710710-a26fde9e-a7ea-415b-9f80-21b2ffd096e3.png)


:hearts: Thank you!
